Description: replace obsolete file() constructor with a context manager.
 Replace file(), specific to python2, with a context manager that
 also closes the file after use.
Author: Nicolas Boulenguez <nicolas@debian.org>

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -48,9 +48,10 @@
 
 def get_version():
     """Extract the version from configure.in"""
-    for line in file("../configure.in").readlines():
-        if line.startswith("AC_INIT"):
-            return line.split(",")[1]
+    with open ("../configure.in") as configure_in:
+        for line in configure_in:
+            if line.startswith("AC_INIT"):
+                return line.split(",")[1]
     raise Exception("Cannot find version number")
 
 # The version info for the project you're documenting, acts as replacement for
